libretroshare/src/rsserver/p3face-server.cc

```diff
@@ -108,21 +106,20 @@ RsServer::RsServer() :
 	msgSrv = NULL;
 	chatSrv = NULL;
 	mStatusSrv = NULL;
-    mGxsTunnels = NULL;
-
-    mMin = 0;
-    mLoop = 0;
-
-
-    mLastts = getCurrentTS();
-    mLastSec = 0; /* for the slower ticked stuff */
-    mTimeDelta = 0.25 ;
+	mGxsTunnels = NULL;
 
-    mAvgTickRate = mTimeDelta;
+	mLastts = getCurrentTS();
+	mTickInterval = maxTickInterval ;
+	mAvgRunDuration = 0;
+	mLastRunDuration = 0;
+	mCycle1 = 0;
```

Comment:
If these are doubles, it's better to initialize them with 0.0
It wont change anything except that when you read you know it's a double.